stammer.py

```diff
@@ -146,34 +146,75 @@ def create_output_audio(best_matches, modulator_audio, carrier_frames, modulator
 
     wavfile.write(TEMP_DIR / 'out.wav', INTERNAL_SAMPLERATE, output_audio)
 
+COMMON_AUDIO_EXTS = [
+    "wav",
+    "wv",
+    "mp3",
+    "m4a",
+    "aac",
+    "ogg",
+    "opus",
+]
+
+def is_audio_filename(name):
+    import os.path
+    return os.path.splitext(name)[1][1:] in COMMON_AUDIO_EXTS
+
+def get_audio_as_wav_bytes(path):
+    import io
```

Comment:
`import` statements...

Reply:
...are something I wish I could've gotten to, but I was short on time :)